cylc/flow/parsec/OrderedDict.py

```diff
@@ -95,12 +93,9 @@ def __bool__(self):
         return bool(list(self.keys()))
 
     def prepend(self, key, value):
-        """Prepend new item in the ordered dict.
-
-        https://stackoverflow.com/questions/16664874/
-           how-can-i-add-an-element-at-the-top-of-an-ordereddict-in-python
-
-        """
+        """Prepend new item in the ordered dict."""
+        # https://stackoverflow.com/questions/16664874/
+        #   how-can-i-add-an-element-at-the-top-of-an-ordereddict-in-python
```

Comment:
I'm confused why are you converting docstrings to comments?

What's the gain.

Reply:
Not much gain, but I figure those don't describe what the methods do, so they're not suitable to be docstrings